README.md

```diff
@@ -206,11 +213,14 @@ that you have a 32bit version of Python installed.
 
 ### Set up your django environment
 
+Before running any of the commands below, you should have all of the following running: couchdb, redis, elasticsearch,
+and kafka. To install and run kafka, follow the instructions here: 
```

Comment:
Like I said I don't remember which of the commands below was failing with out it. I think @esoergel would though, he was helping me when we realized that I needed to run kafka to keep going.